.github/workflows/ci.yml

```diff
@@ -25,8 +25,8 @@ jobs:
 
     - name: Rubocop
       run: |
-        gem install rubocop rubocop-performance --no-document
-        rubocop --require rubocop-performance --format progress
+        bundle install --without test
```

Comment:
This will still install `jruby-openssl`, `rake` and `pry`. Can we install only the `lint` group?

Reply:
No, unfortunately: https://github.com/rubygems/bundler-features/issues/59

Reply:
The I'd keep the `gem install rubocop rubocop-performance --no-document`, unsure how much the extra gems will slow the build down. Particularly worried about `jruby-openssl` and `rake`

Reply:
I've changed to `bundle-only`, [recommended by Bundler member](https://github.com/rubygems/bundler/pull/4907#issuecomment-241675023).

---

Oh, [it's broken](https://github.com/lostisland/faraday/pull/1149/checks?check_run_id=597784850) because of `:git` source for one of gems.

OK, I've changed to manually `with` and `without`. Via config, not deprecated flags:

```
[DEPRECATED] The `--without` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set without 'development test'`, and stop using this flag
[DEPRECATED] The `--with` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set with 'lint'`, and stop using this flag
```

---

> This will still install `jruby-openssl`

No, only for JRuby environment. You can see that there is no installation of any `ssl` gem messages in RuboCop task. I can move it into some group, but I don't know in which one, it look generic.

> `rake` and `pry`

Moved into `development` group.

> Can we install only the lint group?

Now it's done.

> The I'd keep the `gem install rubocop rubocop-performance --no-document`

It's the bad approach! We lock `rubocop` versions in the `Gemfile`, and we need to install these specific version. And duplicate versions (like via `-v` flag for `gem install`) is very bad practice. So, I'm ready to resolve it in correct direction and speed up builds as you want.

---

Bundler flags does not work with bundled Bundler version, so we need to run `gem install bundler` (to update it).